core/src/main/resources/hudson/tasks/ArtifactArchiver/help-artifacts.html

```diff
@@ -5,4 +5,11 @@
    -- except that <tt>","</tt> (<em>comma</em>) is the only supported separator.
    The base directory is <a href='ws/'>the workspace</a>.
    You can only archive files that are located in your workspace.
+
```

Comment:
```suggestion
<br/>
```
Do you have a screenshot of how is generated this help ? I think that if you want to keep space to format you need a <br> tag or maybe a <p> ...</p>. Not sure on what is the best.